db/migrate/20241230153300_new_embeddings_tables.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+class NewEmbeddingsTables < ActiveRecord::Migration[7.2]
+  def up
+    # We need "model_id" to be a bigint now that it's no longer a hardcoded ID.
+
+    create_table :ai_topics_embeddings, id: false do |t|
+      t.bigint :topic_id, null: false
+      t.bigint :model_id, null: false
+      t.integer :model_version, null: false
+      t.integer :strategy_id, null: false
+      t.integer :strategy_version, null: false
+      t.text :digest, null: false
+      t.column :embeddings, "halfvec", null: false
+      t.timestamps
+
+      t.index %i[model_id strategy_id topic_id],
```

Comment:
Where is the index for perf ? 

https://github.com/discourse/discourse-ai/blob/main/db/migrate/20241008054440_create_binary_indexes_for_embeddings.rb

Reply:
I'm replicating stuff from the old table so no particular reason. I imagine is because of `unique: true`